straxen/plugins/peaklets/peaklets.py

```diff
@@ -37,7 +37,7 @@ class Peaklets(strax.Plugin):
     parallel = 'process'
     compressor = 'zstd'
 
-    __version__ = '1.0.1'
+    __version__ = '1.1.0'
```

Comment:
I am not sure about the versioning logic here. Why not `1.0.2`?

Reply:
Agreed and modified.